Solutions/Menes.Abstractions/Menes/IOpenApiLinkOperationMap.cs

```diff
@@ -22,7 +22,7 @@ public interface IOpenApiLinkOperationMap
         /// This should be used where there is a single global operation for links with a given content type and relation type.
         /// </para>
         /// </remarks>
-        void Map(string contentType, string relationType, string operationId);
+        void MapByContentTypeRelationAndOperationId(string contentType, string relationType, string operationId);
```

Comment:
If we're using `Relation` in the method name, should we rename the parameter to `relation` as well?